### ISSUE TITLE
Graphs

### ISSUE THREAD

**Issue by yuriy1337** [yuriy1337/Lunch-Randomizer#40]
Create graphs to shows what places are picked and their frequency.

**Comment by tomarra** [comment]
I think this should be done at the group level and also at the admin level.

**Comment by yuriy1337** [comment]
Agreed. Each group and view their own statistic and admins can view for the whole site & other groups
